src/contexts/language/languageContextBinding.ts

```diff
@@ -113,6 +114,8 @@ const componentGenerator = function* (): Generator<{
   yield { componentDetector: JavaComponentDetector, detectedLanguage: ProgrammingLanguage.Kotlin };
   yield { componentDetector: PythonComponentDetector, detectedLanguage: ProgrammingLanguage.Python };
   yield { componentDetector: GoComponentDetector, detectedLanguage: ProgrammingLanguage.Go };
+  yield { componentDetector: AndroidComponentDetector, detectedLanguage: ProgrammingLanguage.Java };
+  yield { componentDetector: AndroidComponentDetector, detectedLanguage: ProgrammingLanguage.Kotlin };
```

Comment:
I'm getting double warnings, because I'm using Java and Kotlin languages again, I don't know if I'm going good.